torchtitan/parallelisms/parallelize_llama.py

```diff
@@ -146,7 +167,7 @@ def parallelize_llama(
                 enable_compile=job_config.training.compile,
                 enable_compiled_autograd=job_config.experimental.enable_compiled_autograd,
             )
-
+    return model
```

Comment:
IIUC, this is needed because of https://github.com/tianyu-l/torchtitan_intern24/blob/main/train.py#L169

Let's remove the `model = ` over there, and also remove this line

Reply:
but it seems after removing `return model` and the line you pointed, the compiled model is not returned and actual training model is in eager model

I don't know which one you think would be better: (1) keep this `return model` things; (2) do `model.layers.register_module(layer_id, transformer_block)` like fsdp2 compiled mode